gazebo_ros_pkgs/package.xml

```diff
@@ -12,16 +15,18 @@
   <url type="bugtracker">https://github.com/ros-simulation/gazebo_ros_pkgs/issues</url>
   <url type="repository">https://github.com/ros-simulation/gazebo_ros_pkgs</url>
 
-  <author>John Hsu, Nate Koenig, Dave Coleman</author>>
+  <author>John Hsu</author>
+  <author>Nate Koenig</author>
+  <author>Dave Coleman</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
   <exec_depend>gazebo_dev</exec_depend>
   <exec_depend>gazebo_msgs</exec_depend>
-  <exec_depend>gazebo_plugins</exec_depend>
-  <exec_depend>gazebo_ros</exec_depend>
+  <!--exec_depend>gazebo_plugins</exec_depend-->
+  <!--exec_depend>gazebo_ros</exec_depend-->
```

Comment:
```gazebo_ros``` should be okay to depend on now I think

Reply:
oh yeah true it's already there, re-added :wink: